src/styles/LogInPage.css

```diff
@@ -0,0 +1,102 @@
+.login {
+  background-color: #97bf0f;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 16;
+}
+
+.login-wrapper {
+  height: 70%;
+  width: 70%;
+  padding-top: 9%;
+  margin: 10% auto;
+  background-color: #8caa2a;
+}
+
+.login h1 {
+  color: white;
+  text-align: center;
+}
+
+.login-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.login-form input {
+  width: 70%;
+  margin: 0 20% 20px 15%;
+}
+
+.login-form > *::placeholder {
+  font-weight: bolder;
+}
+
+.login-form button {
+  width: 35%;
+  margin: auto;
+  background-color: white;
+  border: none;
+  padding: 5%;
+  color: #97bf0f;
+  border-radius: 20px;
+  font-size: 20px;
+}
+
+button:hover {
+  background-color: rgb(89, 122, 83);
+}
+
+.links {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.links h3,
+.links a {
+  color: white;
+  margin: 3px 0;
+}
+
+.links a:hover {
+  background-color: rgb(89, 122, 83);
+  padding: 10px;
+  border-radius: 25px;
+  text-decoration: none;
+}
+
+@media screen and (min-width: 450px) {
+  .login {
+    height: 100vh;
+  }
+
+  .login-form {
+    margin-top: 5%;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap: 5%;
+  }
+
+  .login-form > * {
+    padding: 1%;
+    flex-basis: 35%;
+    text-align: center;
+  }
+
+  .login-form > *::placeholder {
+    text-align: center;
+  }
+
+  .login-form button {
+    margin-top: 1%;
+    padding: 5px;
+    flex-basis: 30%;
+    cursor: pointer;
+  }
+}
```

Comment:
`OPTIONAL` Please Consider updating the styles for Mobile Viewport. Styles are considered to be one of the key aspects that make the user experience to another level.